clients/src/main/java/org/apache/kafka/common/requests/DescribeQuorumResponse.java

```diff
@@ -72,6 +72,18 @@ public int throttleTimeMs() {
         return DEFAULT_THROTTLE_TIME;
     }
 
+    public static DescribeQuorumResponseData singletonErrorResponse(
+        TopicPartition topicPartition,
+        Errors error
+    ) {
+        return new DescribeQuorumResponseData()
+            .setTopics(Collections.singletonList(new DescribeQuorumResponseData.TopicData()
+                .setTopicName(topicPartition.topic())
+                .setPartitions(Collections.singletonList(new DescribeQuorumResponseData.PartitionData()
+                    .setErrorCode(error.code())))));
```

Comment:
Should we also set the partition index based on `topicPartition` to be complete? It is zero by default so it works here but that could be a nasty bug if the helper is reused in another context.

Reply:
I agree, we should use the partition index from `topicPartition`.

Reply:
Good catch. This was missing in `singletonResponse` as well.